### DISCUSSION TITLE
Search in "Tiles", should search across all tags

### DISCUSSION THREAD

**Discussion by ecker00** [linuxserver/Heimdall#1322] [Ideas]
Currently the Tiles search will only search the current tag you are viewing (like the dashboard), but if you've nested things under other tags, then search will not find it.

Think it would be a lot more intuitive to let the Tile search browse across all applications. It is great that the first search result when hitting enter opens that application, makes it very fast to find things (if it searched all applications).

**Comment by bob454522** [comment]
I agree with this - also the search should be a fuzzy type of search (or ideally the user could configure the type of search).

what i mean by fuzzy:
say i have a title of:  cat mouse dog

if i start to type in the search bar:  cat dog  (i should see the button for "cat mouse dog"  as it is now, you would have to search "cat mouse" - or in other words, currently, the order of the words must be exact).
thanks

**Comment by decentropy** [comment]
yes, im using heimdall on umbrel and search only works across tag names... not item names. thats not very useful... most people will have dozens of item for a smaller set of tags

**Comment by cavo789** [comment]
Hello everyone

Is there any news about this research because, yes, this feature would be really useful...  When I'm on my home page, I'd like to be able to search ‘wherever’ I find what I'm looking for; whether it's on my home page or in other tags.

Ideally, we wouldn't have to open a specific tag first before launching the search because, if we're looking for something, we don't really know where we've put it ;)

Thanks!
